tools/cloud-build/daily-tests/tests/slurm-v5-startup-scripts.yml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+test_name: slurm-gcp-v5-startup-scripts
+deployment_name: "ss-v5-{{ build }}"
+# Manually adding the slurm_cluster_name for use in node names, which filters
+# non-alphanumeric chars and is capped at 10 chars.
+slurm_cluster_name: "ssv5{{ build[0:6] }}"
+zone: us-central1-c
+workspace: /workspace
+blueprint_yaml: "{{ workspace }}/tools/validate_configs/test_configs/hpc-cluster-small-slurm-v5.yaml"
+network: "{{ deployment_name }}-net"
+max_nodes: 5
+# Note: Pattern matching in gcloud only supports 1 wildcard, centv5*-login-* won't work.
+login_node: "{{ slurm_cluster_name }}-login-*"
+controller_node: "{{ slurm_cluster_name }}-controller"
+post_deploy_tests:
+- test-partitions.yml
+custom_vars:
+  partitions:
+  - compute
+  - debug
```

Comment:
We run the startup script on both partitions. Is this adding value. I am wondering if we would get better coverage by running the startup script on 1 partition and having no startup script on the other partition. This way we test both cases. I realize we probably test the no startup script scenario in other tests. Any thoughts, feel free to push back.

Reply:
That makes sense. I also renamed the test to make it more startup-script centric, so even if it's covered elsewhere, testing it here will reasonably be in scope.

Fixed, but rerunning test group 2 to verify everything still works.